RecoLocalTracker/SiPixelRecHits/plugins/SiPixelRecHitSoAFromLegacy.cc

```diff
@@ -25,7 +25,7 @@
 #include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEBase.h"
 #include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEFast.h"
 
-#include "RecoLocalTracker/SiPixelRecHits/plugins/gpuPixelRecHits.h"
+#include "gpuPixelRecHits.h"
```

Comment:
for my education why is this preferred? here and and elsewhere?

Reply:
The idea is that header files in a package's `interface/` directory expose that package's public interface, while header files in a package's `src/` or `plugins/` directories are private to the package.

So, whenever one sees `#include "Some/Package/plugins/header.h"` it looks like a package is including a private header from an other package.
Writing `#include "header.h"` instead makes it immediately clear that it is a header from the local package.